modules/nested-tabs/module.php

```diff
@@ -49,4 +56,32 @@ public function register_styles() {
 			$has_custom_breakpoints ? null : ELEMENTOR_VERSION
 		);
 	}
+
+	/**
+	 * Register script modules.
+	 *
+	 * @return void
+	 */
+	public function register_script_modules() {
+		$this->wordpress_adapter->wp_register_script_module(
```

Comment:
The `wp_register_script_module()` method requires us to register and enqueue the module files directly.